### PR TITLE
Activate tests which are no longer blocked

### PR DESCRIPTION
**Pull request by sutaakar**

### CODE REVIEW

File: test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/survival/DbSurvivalIntegrationTest.java [hunk 1]
```diff
@@ -22,59 +22,54 @@
...
     protected KieServicesClient kieServicesClient;
     protected ProcessServicesClient processServicesClient;
     protected QueryServicesClient queryServicesClient;
     private static final Logger logger = LoggerFactory.getLogger(DbSurvivalIntegrationTest.class);
 
     @Override
-    protected WorkbenchKieServerDatabasePersistentScenario createDeploymentScenario(DeploymentScenarioBuilderFactory deploymentScenarioFactory) {
-        throw new RuntimeException("No deployment scenario with persistent database currently available.");
+    protected KieServerWithDatabaseScenario createDeploymentScenario(DeploymentScenarioBuilderFactory deploymentScenarioFactory) {
+        return deploymentScenarioFactory.getKieServerWithMySqlScenarioBuilder()
```
Comment (jakubschwan) [inline test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/survival/DbSurvivalIntegrationTest.java]: Can you also provide PostgreSQL DB for this test class?

Comment (sutaakar) [reply test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/survival/DbSurvivalIntegrationTest.java]: Sure, will do.

**Comment by sutaakar** [comment]
@jakubschwan Updated and reported the issue :)

**Comment by sutaakar** [comment]
retest this please

**Comment by sutaakar** [comment]
retest this please

**Comment by sutaakar** [comment]
retest this please

**Comment by sutaakar** [comment]
retest this please

**Comment by sutaakar** [comment]
All tests passed  (not sure why they aren't reflected here).

**Pull request merged**